.gitignore

```diff
@@ -5,3 +5,5 @@ lib
 /npm-packages-offline-cache
 tmp
 yarn
+/package-lock.json
```

Comment:
should be fine, there is no reason why you can't `npm link cli-engine` even if cli-engine uses yarn